ocs_ci/templates/ocs-deployment/provider-mode/ocs_storagecluster_converged.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: ocs.openshift.io/v1
+kind: StorageCluster
+metadata:
+  name: ocs-storagecluster
+  namespace: openshift-storage
+spec:
+  allowRemoteStorageConsumers: true
+  arbiter: {}
+  encryption:
+    kms: {}
+  externalStorage: {}
+  flexibleScaling: true
+  hostNetwork: true
+  managedResources:
+    cephBlockPools: {}
+    cephCluster: {}
+    cephConfig: {}
+    cephDashboard: {}
+    cephFilesystems: {}
+    cephNonResilientPools: {}
+    cephObjectStoreUsers: {}
+    cephObjectStores:
+      hostNetwork: false
+    cephRBDMirror: {}
+    cephToolbox: {}
+  mirroring: {}
+  monDataDirHostPath: /var/lib/rook
+  nodeTopologies: {}
+  providerAPIServerServiceType: NodePort
+  storageDeviceSets:
+  - config: {}
+    count: 4
+    dataPVCTemplate:
+      metadata: {}
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 256Gi
+        storageClassName: localblock
+
```

Comment:
Please remove extra line

Reply:
I will work with this yaml more in next commits, will remove this line. Thanks